src/GoodCard/GoodCard.jsx

```diff
@@ -0,0 +1,5 @@
+export const GoodCard = ({ good }) => (
+  <li data-cy="Good">
+    {good}
+  </li>
+);
```

Comment:
whats the point of this component? in this case you can keep it in `GoodsList`, no need to separate it